tests/pytests/test_config.py

```diff
@@ -188,6 +192,7 @@ def test_arg_num(arg_name, arg_value):
     test_arg_num('BG_INDEX_SLEEP_GAP', 15)
     test_arg_num('MINSTEMLEN', 3)
     test_arg_num('INDEX_CURSOR_LIMIT', 128)
+    test_arg_num('BM25STD_TANH_FACTOR', 8)
```

Comment:
Ideally would be as in master (4 instead of 8 - although using `noDefaultModuleArgs` and testing the same behavior)